components/micro-gateway-cli/src/main/resources/templates/service.mustache

```diff
@@ -72,7 +72,13 @@ service {{cut qualifiedServiceName " "}} on {{#api.transport}}{{#equals . "http"
     }
     @gateway:RateLimit{policy : "{{resourceTier}}"}
     resource function {{operationId}} (http:Caller outboundEp, http:Request req) {
-    gateway:checkExpectHeaderPresent(req);
+    if (req.expects100Continue()) {
+        req.removeHeader("Expect");
+        var result = outboundEp->continue();
+        if (result is error) {
+        log:printError("Error sending response", err = result);
```

Comment:
Lets change this error message to add details like error while sending the continue response

Reply:
fixed